src/Panel/Panel.less

```diff
@@ -0,0 +1,45 @@
+@import (reference) "../less/index";
+
+.Panel--container {
+  border-style: solid;
+  border-width: @size_3xs @size_3xs @size_3xs @size_s;
```

Comment:
For the left border, none of the mixins are large enough. Can I use the `@size_s` there or is there a better way to do it.